cfme/utils/testgen.py

```diff
@@ -335,5 +335,4 @@ def pxe_servers(metafunc):
     argnames = ['pxe_name']
 
     items = cfme_data.get('pxe_servers', {}).keys()
-
-    return argnames, [items], items
+    return argnames, items, items
```

Comment:
@jan-zmeskal check all others functions - we expect `argvalues` (in pxe_servers case it's `items`)to be a list.

Reply:
@RonnyPfannschmidt What do you think about this? I think this should not break anything since `cfme_data.get()` returns a list even if it contains only one element and the number of `argnames` is hardcoded.

Reply:
This change causes problems with setup of PXE tests, looking into it

Reply:
we should either align all code in this module or change the way we work with it `return argnames, argvalues, idlist` further. anyway - don't edit `pxe_servers` only

Reply:
@lkhomenk This is not actually something I eventually want to merge. I keep it here more as a reference to @RonnyPfannschmidt